lib/services/dashd.js

```diff
@@ -2250,18 +2250,14 @@ Dash.prototype.getDetailedTransaction = function(txid, callback) {
     for(var outputIndex = 0; outputIndex < result.vout.length; outputIndex++) {
       var out = result.vout[outputIndex];
       tx.outputSatoshis += out.valueSat;
-      var address = null;
-      if (out.scriptPubKey && out.scriptPubKey.addresses && out.scriptPubKey.addresses.length === 1) {
```

Comment:
I think we should leave it for compatibility with older nodes

Reply:
✅